tts_queue_system.py

```
@@ -0,0 +1,425 @@
+            # Wait until it's time to play
+            current_time = time.time()
+            if start_time > current_time:
+                await asyncio.sleep(start_time - current_time)
+            
+            # Play audio
+            message.status = TTSStatus.PLAYING
+            await self._play_audio(audio_data, message)
+            
+            message.status = TTSStatus.COMPLETED
+            self.logger.info(f"Completed message {message.id}")
+            return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)